src/main/java/pp/javastart/budget/BudgetUpdate.java

```diff
@@ -0,0 +1,38 @@
+package pp.javastart.budget;
+
+import java.time.LocalDate;
+import java.util.Scanner;
+
+public class BudgetUpdate {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Podaj id transakcji, którą chcesz zaktualizować");
```

Comment:
Tak, żeby być pro użytkownik to bym jeszcze je wyświetlił, żebym wiedział co mam do dyspozycji :)